src/lib/viz/dataview/formula/FormulaDataViewImpl.ts

```diff
@@ -23,10 +23,13 @@ export class FormulaDataViewImpl extends DataViewImpl<FormulaDataViewData> {
       const aggregatedColumnName = `${this.operation}__${this.column}`;
       const columnName = this.column;
 
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      const values = features.map(
-        (feature: Record<string, unknown>) => feature[aggregatedColumnName] || feature[columnName]
-      );
+      const values = features.map((feature: Record<string, unknown>) => {
+        const aggregatedValue = feature[aggregatedColumnName] as number;
+        const value = feature[columnName] as number;
+        const clusterCount = feature._cdb_feature_count as number;
+
+        return (aggregatedValue || value) * clusterCount;
```

Comment:
@jesusbotella Are you sure about this generic calculation approach for all?, how about 'MIN, MAX, COUNT...'?

We have the VL implementation to check at https://github.com/CartoDB/carto-vl/tree/v1.4.4/src/renderer/viz/expressions/aggregation/viewport, if it's required